src/python/WMCore/__init__.py

```diff
@@ -6,5 +6,5 @@
 
 """
 
-__version__ = '1.1.15.pre6'
+__version__ = '1.1.15.8'
```

Comment:
Where is this version number comes from?

Reply:
I need to switch that back to .pre6.  You have to increment the version every time you upload to pypi.  The change isn't necessary here.